### PR TITLE
Merge in version bump and release notes from Bitcoin Core

### PR DESCRIPTION
**Pull request by rnicoll**

### CODE REVIEW

File: doc/man/dogecoin-qt.1 [hunk 1]
```diff
@@ -72,7 +72,7 @@ Extra transactions to keep in memory for compact block reconstructions
 .HP
 \fB\-par=\fR<n>
 .IP
-Set the number of script verification threads (\fB\-3\fR to 16, 0 = auto, <0 =
+Set the number of script verification threads (\fB\-4\fR to 16, 0 = auto, <0 =
```
Comment (patricklodder) [inline doc/man/dogecoin-qt.1:75]: take this out plz

File: doc/man/dogecoin-cli.1 [hunk 2]
```diff
@@ -1,9 +1,9 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.
-.TH BITCOIN-CLI "1" "June 2017" "dogecoin-cli v0.14.2.0" "User Commands"
+.TH BITCOIN-CLI "1" "June 2017" "dogecoin-cli v1.14.3.0" "User Commands"
 .SH NAME
-dogecoin-cli \- manual page for dogecoin-cli v0.14.2.0
+dogecoin-cli \- manual page for dogecoin-cli v1.14.3.0
 .SH DESCRIPTION
-Dogecoin Core RPC client version v0.14.2.0
+Dogecoin Core RPC client version v0.14.3.0
```
Comment (patricklodder) [inline doc/man/dogecoin-cli.1:6]: wrong version

File: doc/man/dogecoind.1 [hunk 3]
```diff
@@ -77,7 +77,7 @@ Extra transactions to keep in memory for compact block reconstructions
 .HP
 \fB\-par=\fR<n>
 .IP
-Set the number of script verification threads (\fB\-3\fR to 16, 0 = auto, <0 =
+Set the number of script verification threads (\fB\-4\fR to 16, 0 = auto, <0 =
```
Comment (patricklodder) [inline doc/man/dogecoind.1:80]: take this out plz

File: doc/release-notes.md [hunk 4]
```diff
@@ -1,9 +1,9 @@
-Dogecoin Core version 1.14.2 is now available from:
```
Comment (patricklodder) [inline doc/release-notes.md:1]: let's discard all changes to this file and separately do it when we have a full release scope

File: doc/man/dogecoin-cli.1 [hunk 5]
```diff
@@ -1,9 +1,9 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.3.
```
Comment (patricklodder) [inline doc/man/dogecoin-cli.1:1]: hmm it says do not modify?

File: configure.ac [hunk 6]
```diff
@@ -2,7 +2,7 @@ dnl require autoconf 2.60 (AS_ECHO/AS_ECHO_N)
 AC_PREREQ([2.60])
 define(_CLIENT_VERSION_MAJOR, 1)
 define(_CLIENT_VERSION_MINOR, 14)
-define(_CLIENT_VERSION_REVISION, 2)
+define(_CLIENT_VERSION_REVISION, 3)
 define(_CLIENT_VERSION_BUILD, 0)
 define(_CLIENT_VERSION_IS_RELEASE, true)
```
Comment (patricklodder) [inline configure.ac:7]: not IS_RELEASE yet?

**Review by patricklodder — COMMENTED** [review]
needs some work

File: doc/man/dogecoin-cli.1 (hunk 5 above)
Comment (rnicoll) [reply doc/man/dogecoin-cli.1:1]: I believe it means "If you change this, expect it to be overwritten"

Happy that we re-create this with `help2man` befoer release

File: configure.ac (hunk 6 above)
Comment (rnicoll) [inline configure.ac:7]:
```suggestion
define(_CLIENT_VERSION_IS_RELEASE, false)
```

File: doc/man/dogecoin-qt.1 (hunk 1 above)
Comment (rnicoll) [inline doc/man/dogecoin-qt.1:75]:
```suggestion
Set the number of script verification threads (\fB\-3\fR to 16, 0 = auto, <0 =
```

File: doc/man/dogecoind.1 (hunk 3 above)
Comment (rnicoll) [inline doc/man/dogecoind.1:80]:
```suggestion
Set the number of script verification threads (\fB\-3\fR to 16, 0 = auto, <0 =
```

File: doc/man/dogecoin-cli.1 (hunk 5 above)
Comment (patricklodder) [reply doc/man/dogecoin-cli.1:1]: that's ok with me.

**Comment by rnicoll** [comment]
We should re-create the man pages ourselves before release, but this means we're synced with BC 0.14.3 at least.

**Comment by patricklodder** [comment]
> We should re-create the man pages ourselves before release, but this means we're synced with BC 0.14.3 at least.

Are we though? There is no bitcoin source pulled into 1.14.3 except this.

File: doc/release-notes.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
...
+Dogecoin Core version 1.14.3 is now available from:
 
-  <https://github.com/dogecoin/dogecoin/releases/tag/v1.14.0/>
+  <https://github.com/dogecoin/dogecoin/releases/tag/v1.14.3/>
```
Comment (patricklodder) [inline doc/release-notes.md:3]: link does not lead anywhere

**Review by patricklodder — CHANGES_REQUESTED** [review]
ok if remove `doc/release-notes.md` completely and squash into your commit, because every single line is then done by you and not upstream, so it attributes your inspiration at most right now.

**Comment by rnicoll** [comment]
> 
> 
> > We should re-create the man pages ourselves before release, but this means we're synced with BC 0.14.3 at least.
> 
> Are we though? There is no bitcoin source pulled into 1.14.3 except this.

There is - our 1.14.2 was rebased on BC 0.14.3. Check the history, their 0.14.3 commits are _mostly_ in, apart from these three.

File: doc/release-notes.md (hunk 7 above)
Comment (rnicoll) [reply doc/release-notes.md:3]: Cyclic update issue - we need to have link it will be at, in the readme before we create the tag to make the link valid.

**Comment by rnicoll** [comment]
Closed to replace the one fix we need with a stand-alone commit

**Pull request closed without merging**